ntbs-service/Pages/Notifications/NotificationEditModelBase.cs

```diff
@@ -56,6 +56,13 @@ public virtual async Task<IActionResult> OnGetAsync(bool isBeingSubmitted = fals
             }
 
             await AuthorizeAndSetBannerAsync();
+            if (PermissionLevel == PermissionLevel.ReadOnly)
+            {
+                return Notification.NotificationStatus == NotificationStatus.Draft
+                    ? RedirectToPage("/Index")
```

Comment:
I agree that the unauthorized page would be better here

Reply:
I have now made this redirect to the draft overview, which I didn't know we did! (We use it also when you go to a draft with no permissions, so makes sense to use here too)